server/plugin.go

```diff
@@ -42,10 +42,16 @@ type externalConfig struct {
 
 	// Comma separated list of jira groups with permission. Empty is all.
 	GroupsAllowedToEditJiraSubscriptions string
+
+	// Maximum attachment size allowed to be uploaded to Jira, can be a
+	// number, optionally followed by one of [b, kb, mb, gb, tb]
+	MaxAttachmentSize string
```

Comment:
On the server we just accept a size in bytes. This might be better in the UI? Not saying no, it's cool. But maybe over-complicating things.

Reply:
Yeah, well, I am jaded from my past, so I figured I'd add it in. This new type accepts a plain number just as well.

Reply:
I figured this will find its way to the UI sooner or later, and maybe we should port `ByteSize` to server and use it there, too?